settings.gradle.kts

```diff
@@ -23,22 +23,30 @@ dependencyResolutionManagement {
         google()
         mavenCentral()
         maven("https://maven.pkg.github.com/govuk-one-login/mobile-android-ui") {
-            if (file("${rootProject.projectDir.path}/github.properties").exists()) {
-                val propsFile = File("${rootProject.projectDir.path}/github.properties")
-                val props = java.util.Properties().also { it.load(java.io.FileInputStream(propsFile)) }
-                val ghUsername = props["ghUsername"] as String?
-                val ghToken = props["ghToken"] as String?
-
-                credentials {
-                    username = ghUsername
-                    password = ghToken
-                }
-            } else {
-                credentials {
-                    username = System.getenv("USERNAME")
-                    password = System.getenv("TOKEN")
-                }
-            }
+            getCreds(this)
+        }
+        maven("https://maven.pkg.github.com/govuk-one-login/mobile-android-authentication") {
+            getCreds(this)
+        }
+    }
+}
+
+fun getCreds(repo: MavenArtifactRepository) {
+    if (file("${rootProject.projectDir.path}/github.properties").exists()) {
+        val propsFile = File("${rootProject.projectDir.path}/github.properties")
+        val props =
+            java.util.Properties().also { it.load(java.io.FileInputStream(propsFile)) }
+        val ghUsername = props["ghUsername"] as String?
+        val ghToken = props["ghToken"] as String?
+
+        repo.credentials {
+            username = ghUsername
+            password = ghToken
+        }
```

Comment:
Unfortunately, this is considered problematic due to previous occurrences of accidental pushing of the github.properties file. Please confirm with @chrisdondig on the path forward for this, as I know he had written up a tech design.